heroku_connect/db/models/base.py

```diff
@@ -34,6 +58,7 @@ def __new__(mcs, name, bases, attrs):
             new_class._meta.local_fields.remove(is_deleted)
 
         new_class._meta.managed = False
+        new_class.add_to_class('objects', HerokuConnectQuerySet.as_manager())
```

Comment:
I believe this is not the write way to do that, also you did not cover related managers.
See https://docs.djangoproject.com/en/2.0/topics/db/managers/#default-managers